ntirpc/rpc/svc.h

```diff
@@ -155,6 +155,7 @@ typedef struct svc_init_params {
 #define SVC_XPRT_FLAG_DESTROYED		0x0020	/* SVC_DESTROY() was called */
 #define SVC_XPRT_FLAG_DESTROYING	0x0040	/* (*xp_destroy) was called */
 #define SVC_XPRT_FLAG_VSOCK             0x0080
+#define SVC_XPRT_FLAG_ALLOCATED		0x8000
```

Comment:
Belay that. I don't see why this flag is needed. If the pointer is NULL, it isn't allocated.